minitests/partial_reconfig_flow/Makefile

```diff
@@ -0,0 +1,83 @@
+# Top-level target for generating a programmable bitstream.  Given a .fasm
+# file, calling make with the .fasm extension replaced with .hand_crafted.bit
+# will generate a bitstream that includes both the harness and the .fasm design
+# ready for programming to a board.  For example, 'make
+# roi_noninv.hand_crafted.bit' will generate a bitstream that includes the
+# design from roi_noninv.fasm. 
+%.hand_crafted.bit: init_sequence.bit %.no_headers.bit final_sequence.bit
+	cat $^ > $@
+
+%.no_headers.bit: %.patched.bit
```

Comment:
Doesn't this make a .bin instead of a .bit?

Reply:
Good point.  That probably needs to be fixed in a few places.